pkg/planner/core/plan_cost_ver2.go

```diff
@@ -170,31 +169,62 @@ func (p *PhysicalTableScan) GetPlanCostVer2(taskType property.TaskType, option *
 	}
 	hasFullRangeScan := ranger.HasFullRange(p.Ranges, unsignedIntHandle)
 
```

Comment:
Please add the comments for following code, such as:
The following code will compute the cost by two part:
1.  compute the condition cost : seperate three part (tiflash / tiflash with lm / tikv)
2. compute the index cost: seperate three part (xxxxx)